pkgs/servers/search/sonic-server/default.nix

```diff
@@ -21,8 +21,15 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-9XSRb5RB82L72RzRWPJ45AJahkRnLwAL7lI2QFqbeko=";
 
-  # Found argument '--test-threads' which wasn't expected, or isn't valid in this context
-  doCheck = false;
+  checkFlags = [
```

Comment:
We should add a comment why the tests are disabled

Reply:
Hey, sorry for the delay.

I can't really say why these specific one are broken, we probably have to ask who disabled them in the first place (as I'm mostly going from "disable all" to "disable some")